lottie-compose/src/test/java/com/airbnb/lottie/compose/LottieAnimatableImplTest.kt

```diff
@@ -431,11 +430,7 @@ class LottieAnimatableImplTest {
         assertEquals("lastFrameNanos at %d".format(frameTimeMs), lastFrameNanos, anim.lastFrameNanos)
     }
 
-    private fun runTest(test: suspend CoroutineScope.() -> Unit) {
-        runBlockingTest {
-            withContext(clock) {
-                test()
-            }
-        }
+    private fun runTest(test: suspend CoroutineScope.() -> Unit) = kotlinx.coroutines.test.runTest(context = clock + UnconfinedTestDispatcher()) {
```

Comment:
```suggestion
    private fun runTest(test: suspend CoroutineScope.() -> Unit) = runTest(context = clock + UnconfinedTestDispatcher()) {
```

Reply:
This actually is tricky 😅 the _custom method_ is called `runTest` so if I import the one from coroutines it becomes ambiguous which `runTest` does a `runTest { }` invoke. 

As an alternative to fully qualified import I can propose:
1. Introducing an import alias `runTestRaw` or `runTestFromCoroutines`
2. Renaming custom `runTest` to `runTestWithClock`

Reply:
Let's go with number two since it's more explicit and more compatible with the new name